proto/api.proto

```diff
@@ -1405,7 +1408,9 @@ message PostSubmitBatchRequest {
   SubmitStrategy submitStrategy = 2;
   optional bool useBundle = 3;
   optional bool frontRunningProtection = 4;
+  optional google.protobuf.Timestamp timestamp = 5;
```

Comment:
don't think we need this on the entry level

Reply:
this is on the `PostSubmitBatchRequest` struct, the diff on github just makes it look like it is on the entry

Reply:
ok